app/models/game.rb

```diff
@@ -3,4 +3,6 @@ class Game < ApplicationRecord
   has_many :users
   has_many :pieces
   has_many :moves
+
+  scope :available, -> { where("black_player_user_id is null or white_player_user_id is null") }
```

Comment:
Just a minor syntax change:

```suggestion
  scope :available, -> { where("black_player_user_id IS NULL OR white_player_user_id IS NULL") }
```